Protocol.md

```diff
@@ -468,10 +468,18 @@ Frame Contents
     +-----------+-+-+-+-------------+-------------------------------+
     |Frame Type |0|0|R|    Flags    |
     +-----------+-+-+-+-------------+-------------------------------+
-    |0|                  Last Received Position                     |
-    +                                                               +
-    |                                                               |
-    +---------------------------------------------------------------+
+    |0|0|                   Number of Entries                       |
+    +-------------------------------+-------------------------------+
+    |0|                         Stream ID                           |
+    +-------------------------------+-------------------------------+
+    |                         Bytes To Skip                         |
+    +-------------------------------+-------------------------------+
+    |0|                         Stream ID                           |
+    +-------------------------------+-------------------------------+
+    |                         Bytes To Skip                         |
+    +-------------------------------+-------------------------------+
+    |                              ...                              |
+    +-------------------------------+-------------------------------+
```

Comment:
We should also define the meaning of these new fields below.

Reply:
Absolutely, if this draft sounds like a good start, I will finalize it